.github/workflows/ci.yaml

```diff
@@ -613,7 +613,7 @@ jobs:
       - name: Run pylint
         run: |
           . venv/bin/activate
-          pylint homeassistant
+          pylint --init-hook='import sys; sys.path.append("pylint/plugins")' homeassistant
```

Comment:
This is already defined in pyproject.toml?

Reply:
I had trouble getting it to load the module without it.  Let me try it again since I adjusted the `pyproject.toml`.

Reply:
Looks like its not needed anymore.  Good catch 👍